x-pack/plugins/security_solution/public/users/links.ts

```diff
@@ -9,10 +9,15 @@ import { i18n } from '@kbn/i18n';
 import { SecurityPageName, USERS_PATH } from '../../common/constants';
 import { USERS } from '../app/translations';
 import { LinkItem } from '../common/links/types';
+import userPageImg from '../common/images/users_page.png';
 
 export const links: LinkItem = {
   id: SecurityPageName.users,
   title: USERS,
+  image: userPageImg,
+  description: i18n.translate('xpack.securitySolution.appLinks.users.description', {
+    defaultMessage: 'Sudo commands dashboard from the Logs System integration.',
```

Comment:
What does this mean really? Is a user reading this supposed to know what this means? 🤔

Reply:
That is a placeholder. I am waiting for design to provide final descriptions.